src/components/AdminPanel/MedCentersForm/FormPopup/FormPopup.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from 'react';
+import { NavLink, useHistory } from 'react-router-dom';
+import { ROUTS } from '@constants/routs';
+import { useTypesSelector } from '@hooks/UseTypedSelector';
+import Error from '@components/common/Error/Error';
+import { EditPopupWrapper, PopupContent } from './FormPopupStyle';
+
+interface IFormPopupProps {
+  message: string;
+}
+
+const FormPopup: React.FC<IFormPopupProps> = ({ message }) => {
+  const { error } = useTypesSelector((state) => state.medCenter);
+  const history = useHistory();
+  const [seconds, setSeconds] = useState(30);
+
+  useEffect(() => {
+    seconds > 0
+      ? setTimeout(() => setSeconds(seconds - 1), 1000)
+      : history.push(ROUTS.ADMIN_PANEL_MED_CENTERS_LIST);
+  });
```

Comment:
empty line + there should be the dependency array

Reply:
removed this functionality for a while